src/PlaceholderComponent.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import reactStringReplace from 'react-string-replace';
+
+
+function PlaceholderTextComponent({ title }) {
+  return (
+    <React.Fragment>
+      {reactStringReplace(title.text, title.regex, (match, i) => (
+        <span key={i} className={title.cssClass} />
+      ))
+    }
+    </React.Fragment>
+  );
+}
+
+PlaceholderTextComponent.propTypes = {
+  title: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.instanceOf(Object),
+  ]),
+};
+
+PlaceholderTextComponent.defaultProps = {
```

Comment:
I thought there was a default value for regex? The other props don't have default values so can be left out